### ISSUE TITLE
updated_at value should be updated while updating entry

### ISSUE THREAD

**Issue by 13W** [vdemedes/mongorito#83]
please, look
https://github.com/jonschlinkert/set-value/issues/5

**Comment by vdemedes** [comment]
Would be great if you could clarify and provide more details in the issue you referred to, otherwise it's unclear what's wrong.

**Comment by 13W** [comment]
updated_at property should be updated on every update event, e.g:
```
        let session = new Sessions({name: 'test session', date: new Date()});
        yield session.save();
        session = yield Sessions.findOne({_id: session.get('_id')});
        console.log('Original record:');
        console.log(session.get());
        session.set('date', new Date());
        session.set('name', 'updated test session');
        yield session.save();
        session = yield Sessions.findOne({_id: session.get('_id')});
        console.log('Updated record:');
        console.log(session.get());
        Mongorito.disconnect();
```
```
Original record:
{ _id: 56727232b3a6ded22919a39d,
  name: 'test session',
  date: Thu Dec 17 2015 11:28:34 GMT+0300 (MSK),
  created_at: Thu Dec 17 2015 11:28:34 GMT+0300 (MSK),
  updated_at: Thu Dec 17 2015 11:28:34 GMT+0300 (MSK) }
Updated record:
{ _id: 56727232b3a6ded22919a39d,
  name: 'updated test session',
  date: Thu Dec 17 2015 11:28:34 GMT+0300 (MSK),
  created_at: Thu Dec 17 2015 11:28:34 GMT+0300 (MSK),
  updated_at: Thu Dec 17 2015 11:28:34 GMT+0300 (MSK) }
```

date and updated_at in both records are the same, but the name was updated.

This happened while "set-value", which as mongorito dependency, tries to copy Date field instead assigning. bug in this line: https://github.com/jonschlinkert/set-value/blob/master/index.js#L47

**Comment by vdemedes** [comment]
Thanks for reporting this issue! Fix is included on the newly released version on npm - 1.4.6.
